src/Components/App/App.js

```diff
@@ -6,28 +6,50 @@ import Settings from '../Settings/Settings'
 import About from '../About/About'
 import Stats from '../Stats/Stats'
 import PageNotFound from '../PageNotFound/PageNotFound'
-import './App.scss'
 import { enableNotifications } from '../../helpers/notificationHelpers'
-import { ToastContainer } from 'react-toastify';
+import { ToastContainer } from 'react-toastify'
+import './App.scss'
+
 
 function App() {
   enableNotifications()
+
+  const toggleTimerView = (hide) => {
+    const homeContainer = document.getElementById('HomeContainer')
+    if (hide) {
+      homeContainer.classList.add('hidden')
+    } else {
+      homeContainer.classList.remove('hidden')
+    }
+  }
   
   return (
     <section className="App">
-      <ToastContainer 
+      <ToastContainer
         position="top-center"
         autoClose={10000}
         closeOnClick
         pauseOnHover={false}
       />
-      <Header />
+      <Header toggleTimerView={toggleTimerView} />
+      <HomeContainer toggleTimerView={toggleTimerView} />
       <Switch>
-        <Route exact path="/" render={() => <HomeContainer />} />
-        <Route exact path="/settings" render={() => <Settings />} />
-        <Route exact path="/about" render={() => <About />} />
-        <Route exact path="/stats" render={() => <Stats />} />
-        <Route render={() => <PageNotFound />} />
+        <Route exact path="/" />
```

Comment:
Nice call to move HomeContainer out of the Switch statement so it's always rendering!! Makes so much sense now that I see it. Does that not mean you can get rid of the <Route /> on line 37 though?

Reply:
When I got rid of the / route, it went to the 404 as the landing page. I know it seems a little silly to leave the empty route in there, but I think we need it to prevent the 404 from being triggered.

Reply:
Interesting, that only half makes sense to me. 404 routes have never exactly worked how I expected in any React app I've built though!

Reply:
I just learned about Switches in the PHP tutorial I am working through for my job! Cool to see something similar here.